src/Render.cc

```diff
@@ -14,9 +14,9 @@ using namespace snowcrash;
 
 namespace drafter {
 
-    RenderFormat findRenderFormat(std::string contentType) {
+    RenderFormat findRenderFormat(std::string& contentType) {
 
-        if (contentType == "application/json") {
+        if (contentType.find("application/json") != std::string::npos) {
```

Comment:
> I don't think we should do this right now. 

Please elaborate.

also 

> contentType.find("application/json") != std::string::npos 

isn't quite what you want – if I am not mistaken it will match things like `whatchildnot application/json` – you want to `application/json` only at the start of the string.